<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>780</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>720</width>
    <height>680</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>720</width>
    <height>780</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>汉仪文黑-85W</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>721</width>
     <height>771</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>技能定义</string>
    </attribute>
    <widget class="QLabel" name="lb_c">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>命座</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="dsb_enchant">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>60</y>
       <width>70</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>450</y>
       <width>671</width>
       <height>281</height>
      </rect>
     </property>
     <property name="title">
      <string>技能查询</string>
     </property>
     <widget class="QTableWidget" name="tb_skill">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>651</width>
        <height>241</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>名字</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>描述</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>类型</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>30</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>物理攻击附魔比例</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>671</width>
       <height>361</height>
      </rect>
     </property>
     <property name="title">
      <string>技能定义</string>
     </property>
     <widget class="QLineEdit" name="le_a">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>70</y>
        <width>301</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lb_status_q">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>200</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>正确</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_status_e">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>140</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>正确</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_e">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>130</y>
        <width>301</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_q">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>190</y>
        <width>301</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lb_status_a">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>80</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>正确</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="sb_rnd_q">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>190</y>
        <width>46</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="sb_rnd_a">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>70</y>
        <width>46</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="sb_rnd_e">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>130</y>
        <width>46</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>普攻(包括重击）</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>130</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>元素战技</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>190</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>元素爆发</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>40</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>攻击轮数</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>250</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>护盾</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_shld">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>250</y>
        <width>301</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lb_status_shld">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>260</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>正确</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_heal">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>310</y>
        <width>301</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>310</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>治疗</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_status_heal">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>320</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>正确</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="lb_name">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>名字</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_save">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>30</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>保存</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>注释</string>
    </attribute>
    <widget class="QPlainTextEdit" name="pte_cmts">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>671</width>
       <height>661</height>
      </rect>
     </property>
     <property name="plainText">
      <string>自定义注释，技能定义保存时同时保存</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>说明</string>
    </attribute>
    <widget class="QPlainTextEdit" name="plainTextEdit">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>711</width>
       <height>681</height>
      </rect>
     </property>
     <property name="plainText">
      <string>说明
========

物理攻击附魔比例
---------------
物理攻击按比例转成元素攻击，享受本属性伤害加成
其余部分物理攻击，物理伤害加成

技能定义
-------------
定义对应项分技能，全部判定正确可以存储到人物数据文件

规则
-------
1. 可以为 空白 或者 合理的定义
2. 分技能由 + 号连接，如： e1+e2+e3
3. 分技能可带系数，系数可带小数点，系数必须在前, 如 e1+2*e2+3.5*e3
4.普攻/元素战技/爆发 对应分技能必须为 技能查询中类型为 物理伤害、属性元素、环境元素的技能
5.护盾及治疗   对应分技能必须为 技能查询中类型为 技能护盾、治疗、基数的技能
6. e,q,a技能可以出现在 普攻/战技/爆发的任何定义中。 区别是技能等级由首字母定义，伤害加成由位置决定。如公子普攻定义（e1+e2),技能等级随e走，加成享受普攻加成
7.查询表增益类技能不能用于定义
8. 可定义扩散反应，如温蒂大招吃20个属性伤害加7个扩散反应，定义为20*q1+7*ks

技能查询
-------
显示分技能名字，注释及类型，不支持修改

注释
-----
显示自定义注释，技能定义保存时同时保存</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
